Frontend/ui-driver/src/Flow.purs

```diff
@@ -673,14 +678,26 @@ driverDetailsFlow = do
        modifyScreenState $ DriverDetailsScreenStateType (\driverDetailsScreen -> state {props {keyboardModalType = NONE}} )
        homeScreenFlow
 
+    DRIVER_GENDER state -> do
+        let genderSelected = state.data.driverGender
+        _ <- pure $ spy "Gender In FLow" genderSelected
+        let (UpdateDriverInfoReq initialData) = mkUpdateDriverInfoReq
+            requiredData = initialData{gender = genderSelected}
+        (UpdateDriverInfoResp updateDriverResp) <- Remote.updateDriverInfoBT (UpdateDriverInfoReq requiredData)
+        
+        pure $ toast ("Gender Updated")
```

Comment:
@Akash2172 Added translation for the toast message

Remove unnecessary bracket